src/JuliusSweetland.OptiKey/UI/Controls/CKKey.xaml

```diff
@@ -49,7 +49,7 @@
                     <Setter Property="Content">
                         <Setter.Value>
                             <controls:Key SharedSizeGroup="KeyWithText"
-                                          HasImage="True" SymbolGeometry="" 
+                                          SymbolGeometry="" 
```

Comment:
Does the SymbolGeometry have to be set at all?

Reply:
Probably not. CKKey isn't actually used. I've left it in as a placeholder because I would like to use it as a template in CK20Page like how Key is used but my attempts at doing it that way haven't worked so far. It had to be edited because `HasImage` was removed.

If it's too distracting to keep the file I'll delete it.